src/error-code.ts

```diff
@@ -0,0 +1,23 @@
+export interface ErrorCode {
+  code: number;
+  messageText: string;
+}
+
+export namespace ErrorCodes {
```

Comment:
Why a namespace and not an object or enum?

```ts
export const ErrorCodes <Record<string, ErrorCode>>{
    /**
     * ...
     */
    Cannot_perform_arithmetic_on_variables_in_Step_Function: {
      code: 100,
      messageText: "Cannot perform arithmetic on variables in Step Function",
    }
}
``` 

Comments and intellisense seem to work

![image](https://user-images.githubusercontent.com/289213/171449132-bf721413-380a-4c84-8309-2ce97afe27dd.png)

I ask because a namespace creates a functional closure and will bloat the lambda functions and other code. Unless we need to make use of the functional closure and will have more than just error codes. Also less verbose.